homeassistant/components/synology_dsm/__init__.py

```diff
@@ -308,6 +312,10 @@ def _async_setup_api_requests(self):
             self.dsm.reset(self.utilisation)
             self.utilisation = None
 
+        if not self._with_information:
+            self.dsm.reset(self.information)
+            self.information = None
```

Comment:
information API will always be fetched see the lib, remove all use of `_with_information `.

Reply:
According to https://github.com/ProtoThis/python-synology/blob/0.8.2/synology_dsm/synology_dsm.py#L277, it won't be fetched unless set to `True`. Also, if a user decides to disable `temperature` and `uptime`, we doesn't need to fetch `information` anymore

Reply:
I mean, `reset(api)` won't affect information API (see the lib)

Do not set `self.information` to None since it's used in device entry (see `device_info` of this file)

Reply:
Actually Information API is fetched once just after login : https://github.com/ProtoThis/python-synology/blob/0.8.2/synology_dsm/synology_dsm.py#L166-L168

I maintain the lib 😉